GitPrompt.ps1

```diff
@@ -145,13 +145,31 @@ function Write-GitStatus($status) {
     }
 }
 
-if(!(Test-Path Variable:Global:VcsPromptStatuses)) {
-    $Global:VcsPromptStatuses = @()
+$VcsPromptStatusesVariableName = "VcsPromptStatuses"
+
+if (-not (Test-Path Variable:Global:$VcsPromptStatusesVariableName)) {
+    New-Variable -Name $VcsPromptStatusesVariableName -Value @() -Scope Global

```

Comment:
This relates more to the other proposed change, it was so the name of the global variable is controlled & set in one location, currently the $VcsPromptStatusesVariableName variable but potentially it could be set/changed from $GitPromptSettings